WhaleSpotting/react-app/src/styles/PageNav.scss

```diff
@@ -0,0 +1,31 @@
+@import "./constants.scss";
+
+.page-nav {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    font-family: $fontFamily;
```

Comment:
Don't need this

Reply:
we do need 
display: flex;
justify-content: space-between;

removed the others